doc/user-guide/time-series.rst

```diff
@@ -21,20 +21,32 @@ core functionality.
 Creating datetime64 data
 ------------------------
 
-Xarray uses the numpy dtypes ``datetime64[ns]`` and ``timedelta64[ns]`` to
-represent datetime data, which offer vectorized (if sometimes buggy) operations
-with numpy and smooth integration with pandas.
+Xarray uses the numpy dtypes ``datetime64[unit]`` and ``timedelta64[unit]``
+(where unit is anything of "s", "ms", "us" and "ns") to represent datetime
+data, which offer vectorized (if sometimes buggy) operations with numpy and
```

Comment:
Maybe delete `(if sometimes buggy)` now?

The only real bug that I'm familiar with is related to what NumPy considers as the built-in Python scalar corresponding to `ns` precision, which is some form of `int` instead of `datetime.datetime`. This is mitigated significantly by allowing for other datetime64 units.